src/components/CodeEditor/styles.css

```diff
@@ -0,0 +1,4 @@
+:global .CodeMirror {
```

Comment:
For generic components, this should exist in the app and not in the components since these are global styles.